settings.yml

```diff
@@ -41,7 +42,7 @@ speech_to_text:
 # Text to speech
 # ---------------------------
 # This is the default TTS that will be used by Kalliope to talk.
-default_text_to_speech: "pico2wave"
+default_text_to_speech: "googletts"
```

Comment:
Because by default installation of Kalliope with pip doesn't work by default with pico2wave which could be confusing for some new people. Also in Raspbian Buster there are no libttspico-utils and libttspico0 in repositories and must be installed manually from Debian repositores. So since it is a starter pack I would suggest to use default tts/stt which works by default.

Reply:
Actually the sh based installer install pico even on buster. That's why we keep it until now.

Reply:
Ok. I was always installing Kalliope with pip because it was the easiest way for me.

Reply:
Kalliope can't wort without some system packages.
Pip is not enough anyway

Reply:
But since I was more testing it I was able to remember what I needed without checking the documentation so I didn't know that pico is already there. I just created additional PR with changing tts back to pico2wave.